scim.sgnl.yaml

```diff
@@ -0,0 +1,457 @@
+      - name: value
+        externalId: value
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: display
+        externalId: display
+        type: String  
+      - name: type
+        externalId: type
+        type: String
+      - name: primary
+        externalId: primary
+        type: Bool
+  
+  Roles:
+    parent: Users  # External ID of the parent entity
+    displayName: SCIMUserRole
+    externalId: roles
+    description: Entity representing SCIM user roles
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: value
+        externalId: value
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: display
+        externalId: display
+        type: String  
+      - name: type
+        externalId: type
+        type: String
+      - name: primary
+        externalId: primary
+        type: Bool
+
+  Photo:
+    parent: Users  # External ID of the parent entity
+    displayName: SCIMUserPhoto
+    externalId: photos
+    description: Entity representing SCIM user photos
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: value
+        externalId: value
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: display
+        externalId: display
+        type: String  
+      - name: type
+        externalId: type
+        type: String
+      - name: primary
+        externalId: primary
+        type: Bool
+  
+  Certificates:
+    parent: Users  # External ID of the parent entity
+    displayName: SCIMUserCertificate
+    externalId: certificates
+    description: Entity representing SCIM user certificates
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: value
+        externalId: value
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: display
+        externalId: display
+        type: String  
+      - name: type
+        externalId: type
+        type: String
+      - name: primary
+        externalId: primary
+        type: Bool
+
+relationships:
+  # Manager relationship
+  UserManager:
+    name: Manager
+    fromAttribute: Users.$..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].manager.value
+    toAttribute: Users.id
+  
+  # The internal `parent` relationship cannot be directly referenced in `path` relationship.
+  # Create a dummy relationship to represent the `parent` relationship.
+  # The fromAttribute and toAttribute does not matter. Only entities should be specified correctly.
+  # We rely on the `parent` relationship already created during ingestion.
+  DummyParent: 
```

Comment:
I understand the reasoning, but "DummyParent" will throw users off, IMO.
Can we rename it to `UsersParentOf` ?

Reply:
`ParentOfGroupMember`?

Reply:
Ok with that - was thinking of "MemberOf" when I suggested that.